NavigationReactNative/src/BarTitleViewIOS.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { requireNativeComponent, Platform, StyleSheet } from 'react-native';
+
+const NVBarTitleView = requireNativeComponent<any>('NVBarTitleView', null)
+
+class BarTitleView extends React.Component {
+  render() {
+    const { style } = this.props as any
+    return (
+      <NVBarTitleView 
+        {...this.props}
+        style={[styles.barTitleView, style]}>
+        {this.props.children}
+      </NVBarTitleView>
+    )
+  }
+}
+
```

Comment:
We can turn this into a function Component if you prefer?
```jsx
var BarTitleView = ({style, ...props}) => (
  <NVBarTitleView {...this.props} style={[styles.barTitleView, style]} />
);
```

Reply:
Yeah this makes code simpler.